packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx

```diff
@@ -402,16 +402,15 @@ export class GroupedVerticalBarChartBase extends React.Component<
         pointData.color = startColor;
       }
 
+      const gradientId = getId('GVBC_Gradient_ID');
+
       // Not rendering data with 0.
       pointData.data &&
         singleGroup.push(
           <React.Fragment key={`${singleSet.indexNum}-${index}`}>
             {this.props.enableGradient && (
               <defs>
-                <linearGradient
-                  id={`gradient_${index}_${singleSet.indexNum}_${endColor}`}
-                  x1="0%" y1="100%" x2="0%" y2="0%"
-                >
+                <linearGradient id={gradientId} x1="0%" y1="100%" x2="0%" y2="0%">
```

Comment:
use gradient id as prefix only. Append gradient_${index}_${singleSet.indexNum}_${endColor} to it to keep it unique across multiple segments

Reply:
Redefined gradientId as ```getId(<prefix>) + `_${index}_<identifier>_<other identifier>` ``` for all charts